docs/manifest.json

```diff
@@ -2152,17 +2156,15 @@
                           "href": "/docs/references/chrome-extension/add-react-router"
                         },
                         {
-                          "title": "`Sync Host Configuration`",
-                          "wrap": false,
-                          "href": "/docs/references/chrome-extension/sync-host-configuration"
+                          "title": "Sync your Chrome Extension with your web app",
+                          "href": "/docs/references/chrome-extension/sync-host"
```

Comment:
This is probably just an issue with the preview build, but this isn't showing up in the side bar.